lib/pages/dashboard/widgets/timeline/timeline_items.dart

```diff
@@ -107,21 +103,26 @@ class _MatchTimelineItem extends TimelineItemWidget {
 
 class _TimelineItemCard extends StatelessWidget {
   final List<Widget> children;
+  final Function onTap;
 
-  _TimelineItemCard({this.children});
+  _TimelineItemCard({@required this.children, this.onTap});
 
   @override
   Widget build(BuildContext context) {
     return Card(
       elevation: 2.0,
```

Comment:
Can you remove the `elevation` value to be consistent?
Card elevation is set in the theme